src/mumble/AudioConfigDialog.cpp

```diff
@@ -301,8 +297,8 @@ void AudioInputDialog::save() const {
 	s.qsTxAudioCueOff      = qlePushClickPathOff->text();
 
 	s.qsAudioInput    = qcbSystem->currentText();
-	s.bEcho           = qcbEcho->currentIndex() > 0;
-	s.bEchoMulti      = qcbEcho->currentIndex() == 2;
+	s.iEchoOption    = qcbEcho->currentData().toInt();
+	s.bEcho           = s.iEchoOption != ECHO_CANCEL_DISABLED;
```

Comment:
I am wondering whether we could use the current change as an excuse to finally get rid of the 2-step system for the echo cancellation option. Aka: only keep `s.iEchoOption` and get rid of `s.bEcho`. If echo cancellation is disabled, then that should be indicated by `s.iEchoOption == ECHO_CANCEL_DISABLED` :thinking:

Reply:
Yes I think we can. Will do.